x/gov/src/msg/vote.rs

```diff
@@ -0,0 +1,132 @@
+use bytes::Bytes;
+use gears::{
+    core::errors::CoreError,
+    error::IBC_ENCODE_UNWRAP,
+    tendermint::types::proto::Protobuf,
+    types::{address::AccAddress, tx::TxMessage},
+};
+use ibc_proto::google::protobuf::Any;
+use serde::{Deserialize, Serialize};
+
+use super::GovMsg;
+
+mod inner {
+    pub use ibc_proto::cosmos::gov::v1beta1::MsgVote;
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct MsgVote {
+    pub proposal_id: u64,
+    pub voter: AccAddress,
+    pub option: VoteOption,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub enum VoteOption {
+    Empty = 0,
```

Comment:
Please, use common format for all options.

Reply:
Not sure what you mean?

Reply:
It is better to skip byte representation or add it for each option. 

I know that below contains code with this description. Some formal rules improves reading and it can.

Reply:
Added to every option